src/reference/v8.json

```diff
@@ -54,6 +54,20 @@
       "doc": "Default pitch, in degrees. Zero is perpendicular to the surface, for a look straight down at the map, while a greater value like 60 looks ahead towards the horizon. The style pitch will be used only if the map has not been positioned by other means (e.g. map options or user interaction).",
       "example": 50
     },
+    "roll": {
+      "type": "number",
+      "default": 0,
+      "units": "degrees",
+      "doc": "Default roll, in degrees. The roll angle is measured counterclockwise about the camera boresight. The style roll will be used only if the map has not been positioned by other means (e.g. map options or user interaction).",
+      "example": 45,
+      "sdk-support": {
+        "basic functionality": {
+          "js": "5.0.0",
+          "android": "wontfix",
```

Comment:
Why wontfix?

Reply:
You can use https://github.com/maplibre/maplibre-native/issues/2941

Reply:
I will update to link the newly created issue.